skimage/segmentation/__init__.py

```diff
@@ -6,7 +6,7 @@
 from .boundaries import find_boundaries, mark_boundaries
 from ._clear_border import clear_border
 from ._join import join_segmentations, relabel_from_one, relabel_sequential
-from ..morphology import watershed
+from .watershed import watershed
```

Comment:
Let's take this opportunity to rename the file `_watershed`, to avoid confusion between the module and the function.

Reply:
And `_watershed.pyx` -> `_watershed_cy.pyx`.

Reply:
Done